templates/inc/head.tpl

```diff
@@ -1,25 +1,25 @@
 {locale path="../i18n" domain="Alltube"}
-<!Doctype HTML>
+<!doctype html>
```

Comment:
Not sure why this is needed :stuck_out_tongue:

Reply:
Sorry, just used to change it when I'm on a file by convention. 😇
Most of the time it's either `<!doctype html>` (widely used) or `<!DOCTYPE html>` if we want to stick to how the W3C is writing in the specs.
But I can definitely remove this from the PR if needed 👍

Reply:
I guess it makes sense since the rest of the code is in lowercase.
I just wondered if there was some accessibility-related reason.